packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -323,6 +324,7 @@ export default class CFDStore extends BaseStore {
                 this.setError(true, response.error);
                 this.setIsAccountBeingCreated(false);
             }
+            this.root_store.client.setTradersHubLoading(true);
```

Comment:
when do we set this to false?

Reply:
It's resetting the `this.is_landing_company_loaded` state. Initially, this state is set to `false` at the start of the loading process and changes to `true` once everything has loaded.

Reply:
I renamed the function name. Since it was creating confusion.